components/CollectionCells/examples/supplemental/MDCCollectionViewListCell.m

```diff
@@ -0,0 +1,359 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCollectionViewListCell.h"
+#import <MDFInternationalization/MDFInternationalization.h>
+#import "MaterialTypography.h"
+
+static const CGFloat kImagePadding = 16.f;
+static const CGFloat kImageHeight = 40.f;
+static const CGFloat kWithImageRightPadding = 12.f;
+static const CGFloat kTextVerticalPadding = 12.f;
+
+static inline UIFont *defaultTitleFont(void) {
+  return [MDCTypography subheadFont];
+}
+
+static inline UIFont *defaultDetailsFont(void) {
+  return [MDCTypography body1Font];
+}
+
+static inline CGFloat defaultTitleOpacity(void) {
+  return [MDCTypography subheadFontOpacity];
+}
+
+static inline CGFloat defaultDetailsOpacity(void) {
+  return [MDCTypography captionFontOpacity];
+}
```

Comment:
MDCTypography will soon be deprecated in favor of typography schemes — is it possible to avoid using it here?

Reply:
Added the use of MDCTypographyScheme; removed MDCTypography usage from example.